data-esnext.js

```diff
@@ -569,10 +569,11 @@ exports.tests = [
       */},
       res: {
         ie11: false,
-        firefox75: false,
```

Comment:
why is the firefox75 result being removed? manually verified results should never be removed, even if they're redundant.

Reply:
Restored.